### ISSUE TITLE
simpleTokenize does not work

### ISSUE THREAD

**Issue by amir-zeldes** [korpling/pepperModules-RSTModules#2]
The simpleTokenize setting does not seem to be working (V3.0.2). This is the error message:

```
Cannot map 'salt:/0/GUM/GUM_interview_ants' with module 'RSTImporter', because of a mapping result was 'FAILED'.
An exception was thrown by the mapper threads 'Thread[RSTImporter_mapper(salt:/GUM/GUM_interview_ants),5,RSTImporter_mapperGroup]'.
org.corpus_tools.pepper.modules.exceptions.PepperModuleException: Any exception occured while mapping.
        at org.corpus_tools.pepper.impl.PepperMapperControllerImpl.run(PepperMapperControllerImpl.java:189)
Caused by: java.lang.ClassCastException: [Ljava.lang.Object; cannot be cast to [Ljava.lang.Character;
        at org.corpus_tools.peppermodules.rstModules.RST2SaltMapper.mapSegmentsWithTokenize(RST2SaltMapper.java:239)
        at org.corpus_tools.peppermodules.rstModules.RST2SaltMapper.mapSDocument(RST2SaltMapper.java:161)
        at org.corpus_tools.peppermodules.rstModules.RST2SaltMapper.mapSDocument(RST2SaltMapper.java:141)
        at org.corpus_tools.pepper.impl.PepperMapperControllerImpl.map(PepperMapperControllerImpl.java:246)
        at org.corpus_tools.pepper.impl.PepperMapperControllerImpl.run(PepperMapperControllerImpl.java:184)
```

I'm using the RST data from the first document here (ants): https://github.com/amir-zeldes/gum/tree/master/rst

I can use the normal tokenizer, but it mis-tokenizes "M.Sc." and I want to force it to use spaces for tokenization. Here are my import params:

```
...
<importer name="RSTImporter" path="file:/C:/Pepper/corpora/_sandbox/rst/GUM/">
        <customization>
            <property key="rstImporter.tokenize">yes</property>
            <property key="simpleTokenize">' '</property>
        </customization>
</importer>
...
```

**Comment by amir-zeldes** [comment]
The above error is caused by casting the toArray() as Character[] without an explicit `new Character[...]` (since toArray() returns Object). But sadly solving this problem does not fix the bug: the SimpleTokenizer() behaves differently from the TT-Tokenizer and leads to another bug, see korpling/salt#105

**Comment by amir-zeldes** [comment]
The Salt API issue has been resolved in korpling/salt#108. It should be possible to fix the RST Importer now.
